readme.md

```diff
@@ -143,6 +143,7 @@ GitHub Enterprise is also supported. More info in the options.
 - [SVG files in a PR default to rich-diff view.](https://user-images.githubusercontent.com/5243867/57125552-c08a2b00-6d81-11e9-9b84-cdb535baa98e.png)
 - [Download count next to release assets.](https://user-images.githubusercontent.com/14323370/58944460-e1aeb480-874f-11e9-8052-2d4dc794ecab.png)
 - [The most useful comment in issues is highlighted.](https://user-images.githubusercontent.com/1402241/58757449-5b238880-853f-11e9-9526-e86c41a32f00.png)
+- [Add link to forked repo below the source.](https://user-images.githubusercontent.com/55841/60462644-25f77c80-9c4a-11e9-8c69-582d00dd75dd.png)
```

Comment:
* Your repo forks are shown under the repo title. 

The screenshot should be retina

Reply:
Done.